src/Maestro/Maestro.Web/Scripts/tlsconfiguration.bat

```diff
@@ -0,0 +1,8 @@
+SETLOCAL
+SET LOGFILE="%TEMP%\Startup.log"
+SET PollingMode=PollingMode4min
+SET TimeSyncLogFile="%TEMP%\w32time.log"
+
+echo "--- Configuring TLS ---" >> "%LOGFILE%" 2>&1
+POWERSHELL -ExecutionPolicy Unrestricted -File ".\Scripts\Set-TlsConfiguration.ps1" >> "%LOGFILE%" 2>&1
```

Comment:
Since the file _is_ signed, do we have to do Unrestricted? Could it be something more restricted? Or is it not signed by something trusted, so unrestricted is what we need?

Reply:
I think the signature is only valid in the context of the file contents and has no meaning in the context of running it. But I can verify this

Reply:
I got following:
```
Do you want to run software from this untrusted publisher?
File D:\premun\arcade-services\src\Maestro\Maestro.Web\Scripts\Set-TlsConfiguration.ps1 is published by CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington,  C=US and is not trusted on your system. Only run scripts from trusted publishers.
```

Reply:
I can try on Azure and test the VMs and if they have this configured in their trust chain

Reply:
I ran it on the SF node and it did the same - doesn't trust this publisher.